x-pack/plugins/security_solution/public/flyout/left/components/analyze_graph.tsx

```diff
@@ -7,16 +7,57 @@
 
 import type { FC } from 'react';
 import React from 'react';
-import { EuiText } from '@elastic/eui';
+import { EuiEmptyPrompt } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { useLeftPanelContext } from '../context';
 import { ANALYZER_GRAPH_TEST_ID } from './test_ids';
+import { Resolver } from '../../../resolver/view';
+import { useTimelineDataFilters } from '../../../timelines/containers/use_timeline_data_filters';
+import { ERROR_TITLE, ERROR_MESSAGE } from '../../translations';
+import { ALERTS_ACTIONS } from '../../../common/lib/apm/user_actions';
+import { useStartTransaction } from '../../../common/lib/apm/use_start_transaction';
 
 export const ANALYZE_GRAPH_ID = 'analyze_graph';
 
+const ANALYZER = i18n.translate('xpack.securitySolution.flyout.analyzer', {
+  defaultMessage: 'analyzer',
+});
+
 /**
  * Analyzer graph view displayed in the document details expandable flyout left section under the Visualize tab
  */
 export const AnalyzeGraph: FC = () => {
-  return <EuiText data-test-subj={ANALYZER_GRAPH_TEST_ID}>{'Analyzer graph'}</EuiText>;
+  const { eventId } = useLeftPanelContext();
+  const scopeId = 'fly-out';
+  const { from, to, shouldUpdate, selectedPatterns } = useTimelineDataFilters(false); // always false because flyout is not in timeline
+  const { startTransaction } = useStartTransaction();
+
+  if (!eventId) {
+    return (
```

Comment:
I know for now we don't have a storybook for it, but what do you think about creating the `.stories` file anyway similarly to [this one](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/flyout/right/tabs/table_tab.stories.tsx). We'd show the Error state and would also have the TODO linked to the ticket for when we tackle the tech debt